### PR TITLE
update readme file

### PR DESCRIPTION
**Pull request by Pnom**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,2 +1,77 @@
-# Jooby REST server
+# Jooby REST Server
 
+`jooby-rest-server` is a JavaScript application that provides a REST API using the [jooby-codec](https://github.com/jooby-dev/jooby-codec) library. This application allows you to easily decode data.
```
Comment (DarkPark) [inline readme.md]:
```suggestion
It is a JavaScript application that provides a REST API using the [jooby-codec](https://github.com/jooby-dev/jooby-codec) library. This application allows you to easily decode data.
```

File: readme.md [hunk 2]
```diff
@@ -1,2 +1,77 @@
...
+    - [Starting the server](#starting-the-server)
+    - [Running with Docker](#running-with-docker)
+    - [Environment Variables](#environment-variables)
+    - [Examples](#examples)
+      - [Decode](#decode)
+        - [Obis observer](#obis-observer)
+        - [Analog module](#analog-module)
+        - [Mtx lora](#mtx-lora)
+
+## Installation
```
Comment (DarkPark) [inline readme.md]:
```suggestion
## Installation

```

File: readme.md [hunk 3]
```diff
@@ -1,2 +1,77 @@
...
+Make sure you have Node.js and npm installed on your machine. You can install `jooby-rest-server` using the following steps:
+
+```bash
+npm install -g jooby-rest-server
```
Comment (DarkPark) [inline readme.md]:
```
$ npm info jooby-rest-server                                                                                                                                                                                                                                                                                          11:29:45
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/jooby-rest-server - Not found
npm ERR! 404 
npm ERR! 404  'jooby-rest-server@*' is not in this registry.
```

File: readme.md [hunk 4]
```diff
@@ -1,2 +1,77 @@
...
+  --network host \
+  --name jooby-rest-server \
+  joobydev/jooby-rest-server
+```
+
+### Environment Variables
+
+List of available environment variables.
+
+ | name                       | default value   | description                                                                                               |
```
Comment (DarkPark) [inline readme.md]:
```suggestion
| name                       | default value   | description                                                                                               |
```

and all below

File: readme.md [hunk 5]
```diff
@@ -1,2 +1,77 @@
...
+
+
+### Examples
+
+#### Decode
+
+##### Obis observer
+
+```bash
+curl -X POST -H "Content-Type: application/json" -d '{"deviceEUI": "001a79881701b63c", "data": "BAUBCAgAAQ=="}' http://localhost:3000/v1/obis-observer/decoder
```
Comment (DarkPark) [inline readme.md]:
```suggestion
curl -X POST -H "Content-Type: application/json" \
    -d '{"deviceEUI": "001a79881701b63c", "data": "BAUBCAgAAQ=="}' \
    http://localhost:3000/v1/obis-observer/decoder
```
?

File: readme.md (hunk 1 above)
Comment (Pnom) [reply readme.md]: fixed

File: readme.md (hunk 2 above)
Comment (Pnom) [reply readme.md]: fixed

File: readme.md (hunk 3 above)
Comment (Pnom) [reply readme.md]: fixed

File: readme.md (hunk 4 above)
Comment (Pnom) [reply readme.md]: fixed

File: readme.md (hunk 5 above)
Comment (Pnom) [reply readme.md]: fixed

File: readme.md [hunk 6]
```diff
@@ -1,2 +1,79 @@
...
+```bash
+curl -X POST -H "Content-Type: application/json" \
+    -d '{"deviceEUI": "001a79881701b63c", "data": "BAUBCAgAAQ=="}' \
+    http://localhost:3000/v1/obis-observer/decoder
+```
+
+##### Analog module
+
+```bash
+curl -X POST -H "Content-Type: application/json" -d '{"deviceEUI": "001a79881701b63c", "direction": "uplink", "data": "HwICAAtB"}' http://localhost:3000/v1/analog/decoder
```
Comment (DarkPark) [inline readme.md]: would be nice to split this as well (and the next example)

File: readme.md [hunk 7]
```diff
@@ -1,2 +1,79 @@
-# Jooby REST server
+# Jooby REST Server
 
+It is a JavaScript application that provides a REST API using the [jooby-codec](https://github.com/jooby-dev/jooby-codec) library. This application allows you to easily decode data.
+
```
Comment (DarkPark) [inline readme.md:4]:
```suggestion


```

File: readme.md [hunk 8]
```diff
@@ -1,2 +1,79 @@
...
+## Usage
+
+###  Starting the server
+
+```bash
+git clone https://github.com/jooby-dev/jooby-rest-server.git
+cd jooby-rest-server
+npm ci
+npm run start
```
Comment (DarkPark) [inline readme.md]:
```suggestion
npm start
```

?

File: readme.md [hunk 9]
```diff
@@ -1,2 +1,79 @@
...
+
+### Running with Docker
+
+You can also run jooby-rest-server using Docker. Ensure you have Docker installed on your machine, and then use the following command:
+
+```bash
+docker pull joobydev/jooby-rest-server
+
+docker run \
+  --restart unless-stopped \
```
Comment (DarkPark) [inline readme.md]:
```suggestion
    --restart unless-stopped \
```
?

File: readme.md [hunk 10]
```diff
@@ -1,2 +1,79 @@
...
+```bash
+git clone https://github.com/jooby-dev/jooby-rest-server.git
+cd jooby-rest-server
+npm ci
+npm run start
+```
+
+### Running with Docker
+
+You can also run jooby-rest-server using Docker. Ensure you have Docker installed on your machine, and then use the following command:
```
Comment (DarkPark) [inline readme.md]:
```suggestion
You can also run `jooby-rest-server` using Docker. Ensure you have Docker installed on your machine, and then use the following command:
```

File: readme.md [hunk 11]
```diff
@@ -1,2 +1,79 @@
...
+docker run \
+  --restart unless-stopped \
+  --network host \
+  --name jooby-rest-server \
+  joobydev/jooby-rest-server
+```
+
+### Environment Variables
+
+List of available environment variables.
```
Comment (DarkPark) [inline readme.md]:
```suggestion
Available environment variables:
```
?

Comment (Pnom) [reply readme.md]: fixed

File: readme.md (hunk 10 above)
Comment (Pnom) [reply readme.md]: fixed

File: readme.md (hunk 9 above)
Comment (Pnom) [reply readme.md]: fixed

File: readme.md (hunk 8 above)
Comment (Pnom) [reply readme.md]: fixed

File: readme.md (hunk 7 above)
Comment (Pnom) [reply readme.md:4]: fixed

File: readme.md (hunk 6 above)
Comment (Pnom) [reply readme.md]: fixed

File: readme.md (hunk 4 above)
Comment (DarkPark) [reply readme.md]: nope )

File: readme.md (hunk 8 above)
Comment (DarkPark) [reply readme.md]: wrong fix

File: readme.md [hunk 12]
```diff
@@ -1,2 +1,84 @@
...
+
+Available environment variables:
+
+ | name      | default value | description                                                                                               |
+ | --------- | ------------- | --------------------------------------------------------------------------------------------------------- |
+ | NODE_ENV  |               | node environment setup                                                                                    |
+ | LOG_LEVEL | info          | [pino log levels](https://github.com/pinojs/pino/blob/master/docs/api.md#loggerlevel-string-gettersetter) |
+ | HTTP_HOST | `0.0.0.0`     |                                                                                                           |
+ | HTTP_PORT | `3000`        |                                                                                                           |
+
```
Comment (DarkPark) [inline readme.md:56]:
```suggestion
```

Comment (Pnom) [reply readme.md:56]: fixed

File: readme.md (hunk 8 above)
Comment (Pnom) [reply readme.md]: fix again

File: readme.md (hunk 4 above)
Comment (Pnom) [reply readme.md]: fixed now

**Review by DarkPark — APPROVED** [review]

**Pull request merged**